libherokubuildpack/Cargo.toml

```
@@ -0,0 +1,36 @@
+[package]
+name = "libherokubuildpack"
+version = "0.10.0"
+edition = "2021"
+rust-version = "1.60"
+license = "BSD-3-Clause"
+description = "Opinionated common code for buildpacks implemented with libcnb.rs"
+keywords = ["buildpacks", "CNB"]
+repository = "https://github.com/heroku/libcnb.rs"
+documentation = "https://docs.rs/libherokubuildpack"
+readme = "./README.md"
+include = ["src/**/*", "../LICENSE", "../README.md"]
+
```

This will ensure the docs.rs build enables all features, so they are documented (we should probably do this for the `libcnb` crate too, so the anyhow error `From` implementation shows up):

```suggestion

[package.metadata.docs.rs]
all-features = true

```

---

This is a good one, I wasn't even aware that this is a thing we need to do. Thanks! 👍🏻